models/user.mjs

```diff
@@ -1,9 +1,19 @@
 import mongoose from 'mongoose';
 
 const UserSchema = new mongoose.Schema({
-  name: {
-    type: String,
-    required: true,
+  google: {
```

Comment:
This should not be saved here. User model should not be dependent on auth provider.
We should just bootstrap the basic model data on the first login. Email should be the primary attribute.